src/test/java/org/simulator/fba/BiGGTest.java

```diff
@@ -27,26 +28,30 @@
 public class BiGGTest {
 	private String resource;
     private static final Logger logger = LoggerFactory.getLogger(TestUtils.class);
+    private static final double RESULT_DEVIATION = 1E-6d;
 
+    /**
+     * HashMap for storing the reference results with key as the model names.
```

Comment:
I doubt that we need to explicitly show, that HashMap will be stored in this field. Also, one can figure out that it's about the reference results from the name of the field.